vhdbuilder/packer/components.json

```diff
@@ -78,7 +78,7 @@
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "versions": [
-        "v1.4.9"
+        "v1.4.12"
```

Comment:
Can you keep the old v1.4.9 too ? if we have any problem with 1.4.12 we should be able to roll back to 1.4.9

Reply:
ditto I can merge once you add that back. For a period of time both will be used.